Makefile

```diff
@@ -130,7 +130,8 @@ build-mockgen:
 .PHONY: mocks
 mocks: build-mockgen
 	$(call create_mock,pkg/client/introduce,Provider)
-	$(call create_mock,pkg/didcomm/protocol/introduce,InvitationEnvelope)
+	$(call create_mock,pkg/didcomm/protocol/introduce,Provider;InvitationEnvelope)
```

Comment:
we need to find way to clean mock file

Reply:
currently we have mock folder in each sub package

Reply:
sorry not sure I understand, I did not commit mock files, we only generate it on CI and local development (to run the tests). My point is to not commit mocks in the repository.

Reply:
@soluchok make clean should ideally remove all generated and build artifacts.

Reply:
How about to replace it with `gomock` ? Or another name that is not so generic

Reply:
I thought all mocks are located here `aries-framework-go/pkg/internal/mock`

Reply:
@fqutishat @troyronda @rolsonquadras ^

Reply:
Done!

Reply:
looks good just add clean-mocks to clean target